src/poliastro/iod/vallado.py

```diff
@@ -23,6 +23,10 @@ def lambert(k, r0, r, tof, short=True, numiter=35, rtol=1e-8):
         Final position (km).
     tof : ~astropy.units.Quantity
         Time of flight (s).
+    M : int, optional
```

Comment:
```suggestion
    num_rev : int, optional
```

Reply:
Thanks!, I'm saving docstring tests for last. I'll fix in next push